src/components/RichTextEditor/serializer/markdown/isValueEmpty.ts

```diff
@@ -0,0 +1,10 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { SlateToMarkdown, Transform } from '..';
+import { TreeOfNodes } from '../../types';
+
+export const isValueEmpty = (value: TreeOfNodes): Promise<boolean> =>
+    new Promise((resolve) => {
+        const result = Transform.use<TreeOfNodes, string>(new SlateToMarkdown()).process(value);
+        resolve(result.length === 0);
```

Comment:
Can't you check if there is an array of one node and if first node is empty?
It would be way faster

Reply:
Updated with new Function :)